Automattic/wp-calypso
# Notices: remove site-notice component

Part of notices refactor intitiative:
#888 #1094 #1105 #1313 

This component was only used once and didn't add much additional functionality.

Plus, design-wise, after clicking domain name, the user was redirected to domain management, which was kind of confusing.

# Design changes

Before:
![](https://cldup.com/8wxRw4QRsk-3000x3000.png)

After:
![](https://cldup.com/wdl2Fl71ej-1200x1200.png)

### Why would you put so much text there?
This is actually the most important information in the whole site right now.
I am wondering if it should be put in a permanent global notice, because no matter what people change in Calypso, they wont see anything on the frontend.

And when I was searching for a way to turn off redirection, I had to dig around A LOT.
@mtias: Any suggestions how to make this look good?

# Testing

1. Set site redirect
2. Open sidebar
3. Notice the notice (get it?)
